tests/conftest.py

```diff
@@ -12,13 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
 import os
+import shutil
 import tempfile
 from subprocess import Popen
 from typing import List
 
 import pytest
 from tests.helpers import delete_file
+from tests.utils.torch import find_file_with_pattern
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+try:
```

Comment:
is there any better place to put this outside of conftest? would prefer to keep closer to the place where it's used